### PR TITLE
Adjust the topology of the global trade in the cim scenario.

### PR DESCRIPTION
**Pull request by wesley-stone**
# Description

## Linked issue(s)/Pull request(s)

- [issue_number](issue_link)

## Type of Change

- [ ] Non-breaking bug fix
- [ ] Breaking bug fix
- [x] New feature
- [ ] Test
- [ ] Doc update
- [ ] Docker update

## Related Component

- [x] Simulation toolkit
- [ ] RL toolkit
- [ ] Distributed toolkit

## Has Been Tested

- OS:
  - [x] Windows
  - [x] Mac OS
  - [x] Linux
- Python version:
  - [x] 3.6
  - [x] 3.7
- Key information snapshot(s):
  
## Needs Follow Up Actions

- [ ] New release package
- [ ] New docker image

## Checklist

- [ ] Add/update the related comments
- [ ] Add/update the related tests
- [ ] Add/update the related documentations
- [ ] Update the dependent downstream modules usage

### CODE REVIEW

**Comment by Jinyu-W** [comment]
Update the results of ``no repositioning`` and ``random repositioning`` in ``docs/source/scenarios/container_inventory_management.rst`` together in this PR.

File: maro/simulator/scenarios/cim/topologies/global_trade.22p_l0.8/config.yml [hunk 1]
```diff
@@ -1,234 +1,230 @@
-seed: 4096
```
Comment (ArthurJiang) [inline maro/simulator/scenarios/cim/topologies/global_trade.22p_l0.8/config.yml:1]: @chaosddp @Jinyu-W  do we need this?

File: docs/source/scenarios/container_inventory_management.rst [hunk 2]
```diff
@@ -483,41 +483,41 @@ Random Repositioning
...
+     - 755,990 :math:`\pm`  35800
    * - global_trade.22p_l0.8
-     - 2,241,716
-     - 1,154,184 :math:`\pm`  7,043
-     - 270,960 :math:`\pm`   2,240
+     - 897,716
+     - 216,480 :math:`\pm`  13,768
+     - 743,825 :math:`\pm`   41,461
 
 Quick Start
```
Comment (ArthurJiang) [inline docs/source/scenarios/container_inventory_management.rst:522]: @ysqyang @wesley-stone please also add the experiment result for MDQN and GraphA2C, and also add the example link near to the experiment result table.

File: maro/simulator/scenarios/cim/topologies/global_trade.22p_l0.8/config.yml (hunk 1 above)
Comment (ArthurJiang) [reply maro/simulator/scenarios/cim/topologies/global_trade.22p_l0.8/config.yml:1]: ignore it, looks like the generated config file doesn't keep the order. Please double confirm it.

**Comment by ArthurJiang** [comment]
@wesley-stone add a motivation for this pr, which describes why we update the global trade topology.

**Comment by ArthurJiang** [comment]
And if you have an issue with it, please link it. @wesley-stone

File: docs/source/scenarios/container_inventory_management.rst (hunk 2 above)
Comment (wesley-stone) [reply docs/source/scenarios/container_inventory_management.rst:522]: Currently there is no GraphA2C implementation in this repo. Maybe MDQN is enough.

Comment (wesley-stone) [reply docs/source/scenarios/container_inventory_management.rst:570]: I have added GNN results.

**Comment by Ahmedest61** [comment]
@wesley-stone This branch does not work on global scenario, right? I'm trying to integrate the latest **data_lib** and **simulator** part of Master branch. The Master's `business_engine` does register with `on_arrival` which I think this branch's `business_engine` doing something similar in `step`. Can you share your opinion on this. Thanks!

**Comment by Jinyu-W** [comment]
Out of date

**Pull request closed without merging**